diff -p -up daemon-1.0.1/src/native/unix/native/Makefile.in.asneeded daemon-1.0.1/src/native/unix/native/Makefile.in
--- daemon-1.0.1/src/native/unix/native/Makefile.in.asneeded	2005-04-18 04:32:38.000000000 +0200
+++ daemon-1.0.1/src/native/unix/native/Makefile.in	2009-09-25 22:36:59.000000000 +0200
@@ -36,7 +36,7 @@ libservice.a: $(OBJS)
 	$(RANLIB) libservice.a
 
 jsvc: jsvc-unix.o libservice.a
-	$(LDCMD) $(LDFLAGS) jsvc-unix.o libservice.a -o ../jsvc
+	$(LDCMD) jsvc-unix.o libservice.a $(LDFLAGS) -o ../jsvc
 
 clean:
 	rm -f $(OBJS) ../jsvc jsvc-unix.o libservice.a
